knative/eventing-operator
# Remove obsolete resources caused by merging controllers

# Issue to be fixed

Fixes #90

## Proposed Changes

* Remove old resources that are obsolote after https://github.com/knative/eventing/pull/2448

## Release Note



```release-note
NONE
```

More info on how to find what went obsolete is available at https://github.com/knative/eventing-operator/issues/90

How to verify:
- Install the 0.12.0 operator, create a CR
- replace that operator with this PR
- the obsolote resources should be deleted, new resources should be created

Notes:
- Replaced the `0.12.0.yaml` in `kodata` with the nightly release file (`v20200206-05bba2be.yaml`)
- That nightly release file is fetched like this: `gsutil cp gs://knative-nightly/eventing/latest/eventing.yaml ./`
